examples/browserify-gulp-example/package.json

```diff
@@ -7,21 +7,25 @@
     "url": "https://github.com/callemall/material-ui.git"
   },
   "scripts": {
-    "start": "gulp"
+    "start": "gulp",
+    "lint": "eslint --ext .js src && echo \"eslint: no lint errors\""
```

Comment:
Don't need the `--ext .js` unless we're including other types too (such as `jsx`)